dev/com.ibm.ws.security.social/src/com/ibm/ws/security/social/tai/TAIUserApiUtils.java

```diff
@@ -104,8 +105,9 @@ private synchronized void initializeCache(SocialLoginConfig config) {
         }
     }
 
-    private void cacheUserApiResponse(@Sensitive String serviceAccountToken, String userApiResponse) {
-        userApiCache.put(serviceAccountToken, userApiResponse);
+    private void cacheUserApiResponse(SocialLoginConfig config, @Sensitive String serviceAccountToken, String userApiResponse) {
+        UserApiCacheKey key = new UserApiCacheKey(serviceAccountToken, config.getUniqueId());
+        userApiCache.put(key, userApiResponse);
```

Comment:
It appears that some SocialLoginConfig's have a clock skew.  For those ones that do should there be a call to the put method that pass in the clock skew?